### PR TITLE
Update using authorization to reflect subscription authorization

### PR DESCRIPTION
**Pull request by ferbncode**
> Zalando ticket : [ARUHA-2464](https://jira.zalando.net/browse/ARUHA-2464)

## Description
Update documentation for subscription authorization.

## Review
- [ ] Tests
- [ ] Documentation

## Deployment Notes
These should highlight any db migrations, feature toggles, etc.

### CODE REVIEW

**Comment by rcillo** [comment]
👍

File: docs/_documentation/using_authorization.md [hunk 1]
```diff
@@ -40,10 +39,22 @@ In this section, the `admins` list includes the attributes that authorize a subj
 the `readers` list includes the attributes that authorize a subject to perform the `read` operation; and the `writers` 
 list includes the attributes that authorize a subject to perform the `write` operation;
 
-Whenever an event type is created, or its authorization section is updated, all attributes are validated. The exact 
+Similarly, protecting a subscription can be done during it's creation or as an update to the subscription.
+Authorization section for a subscription looks like below, and contains only the `admins` and the `readers` lists. The
+list of admins specifies the attributes that can do admin operations like deletion or updating a subscription and i
```
Comment (lmontrieux) [inline docs/_documentation/using_authorization.md]: unnecessary `i` at the end of this line

File: docs/_documentation/using_authorization.md [hunk 2]
```diff
@@ -40,10 +39,22 @@ In this section, the `admins` list includes the attributes that authorize a subj
 the `readers` list includes the attributes that authorize a subject to perform the `read` operation; and the `writers` 
 list includes the attributes that authorize a subject to perform the `write` operation;
 
-Whenever an event type is created, or its authorization section is updated, all attributes are validated. The exact 
+Similarly, protecting a subscription can be done during it's creation or as an update to the subscription.
```
Comment (lmontrieux) [inline docs/_documentation/using_authorization.md]: `it's` -> `its`

File: docs/_documentation/using_authorization.md [hunk 3]
```diff
@@ -74,10 +85,24 @@ curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/event-t
 }'
 ```
 
+For creating a subscription, the sample request is as follows:
+
+```bash
+curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/subscriptions/ea2d7472-ddc6-4b9e-91f1-5bcd0b7b4fa4 -d '{
```
Comment (lmontrieux) [inline docs/_documentation/using_authorization.md]: Why is the subscription id in the URL, if this is the creation of a subscription?

Comment (ferbncode) [reply docs/_documentation/using_authorization.md]: Oops, messed up. Updated in `7265d6e`.

**Comment by lmontrieux** [comment]
:+1:

**Comment by ferbncode** [comment]
:+1:

**Pull request merged**